roles/api-server/defaults/main/main.yaml

```diff
@@ -44,3 +44,17 @@ api_server_hpa_max: 5
 api_server_hpa_cpu_percentage: 70
 
 api_server_resources:
+
+api_server_mail_provider: none
```

Comment:
Documentation for these properties would be nice

Reply:
Created an issue to not forget this topic: https://github.com/metal-stack-cloud/ansible-roles/issues/17